content/open-source/vex/what-is-openvex.md

```diff
@@ -0,0 +1,152 @@
+
+To address this problem, SBOM working groups came up with the idea of the Vulnerability Exploitability eXchange (VEX), a system designed to help users eliminate vulnerability “noise” and focus on vulnerabilities that present a genuine risk to their organization. VEX is a mechanism that enables software suppliers and third party stakeholders (such as security researchers) to publish machine-readable security advisories on the status of Common Vulnerabilities and Exposures. When publishing a VEX document for a known vulnerability of a software product, the author assigns the product a status drawn from the following list: 
+
+* NOT AFFECTED – No remediation is required regarding this vulnerability. 
+* AFFECTED – Actions are recommended to remediate or address this vulnerability. 
+* FIXED – These product versions contain a fix for the vulnerability. 
+* UNDER INVESTIGATION – It is not yet known whether these product versions are affected by the vulnerability. An update will be provided in a later release. 
+
+Once published, users of the software product (such as operators or developers) can use these VEX documents to determine whether they are impacted by a vulnerability and what steps they might need to take to address it. Though VEX documents do not need to be used with an SBOM, together they offer a powerful and efficient way to comprehensively scan your codebase for vulnerabilities that matter. An SBOM helps you know whether you have dependencies associated with a vulnerability, while a VEX document tells you which of those vulnerabilities you can ignore. A further benefit of VEX documents is that they are machine readable, enabling users to integrate them into automated workflows and broader tooling to support efficient security management. 
```

Comment:
If "software product" is common in the language for SBOMs then you should stet

Here, maybe I would change "the software product" to "a given software"

Reply:
Also, this explanation of "users" (operators or developers) should go up higher where I complained about the "user" word 😂

Reply:
Software product is used in vex documentation, but I agree it's confusing here. Adjusted the wording. 

Also adjusting use of "users" in this guide.